src/assets/scss/_text-sizes.scss

```diff
@@ -0,0 +1,19 @@
+.alg-h1 {
+  font-size: 3rem;
+}
+
+.alg-h2 {
+  font-size: 2.5rem;
+}
+
+.alg-h3 {
+  font-size: 2rem;
+}
+
+.alg-h4 {
+  font-size: 1.5rem;
+}
+
+.alg-h5 {
+  font-size: 1.25rem;
+}
```

Comment:
> >  `<p class="item-title alg-h1" data-cy="item-title">{{ itemData.item.string.title }}</p>` -> why not using a h1 directly instead of p with a h1 tag? (would be more "semantic")
>
>  Could be, I’ll change

Reply:
Fixed #1362